src/DrCabal/Profile.hs

```diff
@@ -146,10 +156,11 @@ formatChart start end width libs = unlines $ concat $
     legend :: [Text]
     legend =
         [ b "Legend"
-        , "  " <> fmt [cyan]   block <> "  Downloading"
-        , "  " <> fmt [blue]   block <> "  Starting"
-        , "  " <> fmt [red]    block <> "  Building"
-        , "  " <> fmt [yellow] block <> "  Installing"
+        , "  " <> fmt [cyan]    block <> "  Downloading"
+        , "  " <> fmt [blue]    block <> "  Starting"
+        , "  " <> fmt [red]     block <> "  Building"
+        , "  " <> fmt [magenta] block <> "  Haddock"
```

Comment:
Let's use `green` colour for the documentation 🙂 

I believe, it should look different from the `Installing` yellow phase but if they look too similar, `magenta` is fine 👍🏻